__tests__/ActionTrigger.test.ts

```diff
@@ -12,27 +13,27 @@ export class ActionTrigger extends Reactor {
     // This action is never scheduled. It should never be present.
     a2: Action<string> = new Action<string>(this, Origin.logical);
 
-    constructor(parent:Reactor|null) {
+    constructor (parent:Reactor|null) {
```

Comment:
All the files have already been formatted, and `eslint` should not introduce new formatting rules. I don't think this looks particularly good, either. Is there a way to ignore or turn off these suggestions for spacing?

Reply:
Hmm...... This is weird, because aeb72fb (`eslint-config-prettier`) should've disabled all the rules that conflict with `prettier`. But I will run `prettier` when other changes are done.

Reply:
It looks like this added space is still in the diff. Also, I'd like to be assured that there is no ping-pong game going on between `eslint` and `prettier` where they both want a different thing....

Reply:
It looks like this would be the culprit: https://eslint.org/docs/latest/rules/space-before-function-paren

Reply:
I see what's happening: I think the issue is only happening in `__tests__` and I only fixed relevant issues in `src`. Will push a fix for this now.

> I'd like to be assured that there is no ping-pong game 

Indeed, I realised this was a poor decision on my end, so this came into play:

https://github.com/lf-lang/reactor-ts/blob/174c3177d7ccbc169aefd0ddf27bd5fd19ad8a28/.eslintrc.js#L7

This package basically disables all rules that could potentially conflict with `prettier`, including the one we are discussing.
https://github.com/prettier/eslint-config-prettier/blob/311351ba4d60789fa5d7af5bacd0e6a8dc3c7224/index.js#L99

Reply:
That sounds right, but the "space-before-function-paren" rule seems to have still been applied. The line that this discussion is anchored to still has it, for instance.